libs/types/apollo.config.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   client: {
     service: {
       name: 'vega',
-      url: process.env.NX_VEGA_URL,
+      url: 'https://n03.stagnet2.vega.xyz/query',
```

Comment:
question (blocking): this looks like it was left there by accident? not 100% sure